### ISSUE TITLE
Improve event handling identification

### ISSUE THREAD

**Issue by razvancrainea** [OpenSIPS/opensips#1533]
Currently each `event_*` module needs to register a hard-coded unique flag to the event interface in order to identify its sockets. Since this event is hard-coded, this method is very prone to errors (see [this](http://lists.opensips.org/pipermail/users/2018-November/040253.html) thread and 8f73149), we need a way to improve the way we allocate these ids.

Thanks go to SamyGo on the mailing list for reporting this!

**Comment by goharahmed** [comment]
Hi there,
An improvement is a long term solution, any possible value of that flag (as you recommend) to use in current situation to get this working?
Thanks

**Comment by razvancrainea** [comment]
@goharahmed somehow my initial commit SHA was pointing to a different commit.
8bbc185888730fc1c49a8b3f3fec1550afbf4a74 should have already solve this issue, but only temporarily.

**Comment by vitalikvoip** [comment]
(1<<25) is already busy by the event_routing module. The closest free flag is <<23.

**Comment by razvancrainea** [comment]
Yeah, this happened again, 25 was already overlapping with event_flatstore. This definitely needs to move to an error prone mechanism, I am increasing its priority.
